nocodb/docker-compose.yml

```
@@ -0,0 +1,35 @@
+version: "3.7"
+
+services:
+
+  nocodb:
```

Rename service to app.
```suggestion
  app:
```